playbooks/maas-host-cdm.yml

```diff
@@ -48,6 +48,14 @@
         - deploy_osp | bool
 
   tasks:
+    - name: Discover disk device facts
+      gather_disk_device_facts:
+        hostname: "{{ ansible_hostname }}"
+        container_name: "{{ container_name | default(inventory_hostname) }}"
```

Comment:
will this module return anything? if so, do we need a "register" ?

Reply:
The module technically won't return anything, but set a fact on the host running it:

https://github.com/rcbops/rpc-maas/pull/617/files#diff-6cd182564ec970218f86b4c5d2626820R60